src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -101,17 +96,38 @@ impl VirtSandbox {
     #[instrument]
     async fn prepare_for_start_sandbox(
         &self,
-        _id: &str,
+        id: &str,
         network_env: SandboxNetworkEnv,
     ) -> Result<Vec<ResourceConfig>> {
         let mut resource_configs = vec![];
         if !network_env.network_created {
-            if let Some(netns_path) = network_env.netns {
-                let network_config = ResourceConfig::Network(
-                    self.prepare_network_config(netns_path, network_env.network_created)
-                        .await,
-                );
-                resource_configs.push(network_config);
+            let config = self.resource_manager.config().await;
```

Comment:
suggest putting line 104 to line 130 into a new function, it's better for writing UT in the future.

Reply:
Fixed